src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.key);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.key);
+  }
+
+  key = (event: KeyboardEvent) => {
```

Comment:
```suggestion
  keypressHandler = (event: KeyboardEvent) => {
```

`key` is a confusing function name